.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: generic
+language: crystal
```

Comment:
I recommend adding the  redis service to dockercompose.yml and reenabling docker for running our tests.  This will eliminate failures because of differences between environments and gives us finer control of the testing environment.